codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -216,15 +218,14 @@ class ServerServiceGeneratorV2(
             pub fn build(self) -> Result<$serviceName<#{SmithyHttpServer}::routing::Route<$builderBodyGenericTypeName>>, MissingOperationsError>
             {
                 let router = {
-                    use #{SmithyHttpServer}::operation::OperationShape;
+                    ##[allow(unused_mut)]
```

Comment:
Can we add this conditionally on the fact that we know there are no operations?

Reply:
Loosening lints in all cases can lead to bugs going undetected (because a legitimate lint doesn't trigger). If it's easy enough to be precise, I think we should be.

Reply:
> If it's easy enough to be precise, I think we should be.

+1 to this. I don't think we should bend over backwards to have lints be as tight as possible, but if it's easy I'd go for it.